src/components/Typography/src/Heading1.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import BaseProps from '@gemeente-denhaag/baseprops';
+
+export type Heading1Props = BaseProps;
+
+export const Heading1: React.FC<Heading1Props> = (props: Heading1Props) => {
```

Comment:
Not sure we should use numbers in component names. Either write the numbers as words (`HeadingOne`, `HeadingTwo`, et cetera) or perhaps use a different alias? Maybe something like the Greek alphabet (`HeadingAlpha`, `HeadingBeta`)?

Reply:
I have never heard of someone who was confused by this, or anyone who upgraded their `<h1>` to `<h6>` to have the latest features.

Reply:
Agreed with @Robbert. This Using Heading[1-6] perfectly reflects the actual HTML element. And makes any design tokens names easy to understand.